<!DOCTYPE HTML>
<!--
	Hielo by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Research Projects</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="subpage">

		<!-- Header -->
			<header id="header">
				<div class="logo"><a href="index.html">Kroncke Laboratory</a></div>
				<a href="#menu">Menu</a>
			</header>

			<!-- Nav -->
				<nav id="menu">
					<ul class="links">
						<li><a href="index.html">Home</a></li>
						<li><a href="ResearchProjects.html">Research Projects</a></li>
						<li><a href="ResearchProjects.html">People</a></li>
						<li><a href="ResearchProjects.html">Publications</a></li>
						<li><a href="ResearchProjects.html">Resources</a></li>
						<li><a href="ResearchProjects.html">Open Positions</a></li>
					</ul>
				</nav>

		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>How do we determine the significance of genetic variants?</p>
						<h2>Research Program</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
			<section id="two" class="wrapper style2">
				<div class="inner">
					<div class="box">
						<div class="content">
							<header class="align-center">
								<h2>What is the significance of genetic variants to those that carry them?</h2>
							</header>
							<p>The increasing use of next generation sequencing in the clinical arena is uncovering a large number of variants across all genes, but methods to predict their pathogenicity are not well-established. This results in an increasing number of “variants of uncertain significance” (VUSs), a major emerging problem in genomic medicine. Rare variants (mutations) in the cardiac sodium channel gene SCN5A are implicated in diverse heart diseases, including, type 3 long QT syndrome (LQT3) and Brugada syndrome (BrS1), but are also common in healthy populations. While multiple algorithms predict whether SCN5A variants are deleterious (SIFT, PolyPhen-2, PredSNP, CADD, etc.), too many neutral variants are classified as disease-causing. My long-term research interest is to improve our understanding of the clinical burden of ion channel non-synonymous single nucleotide polymorphisms (nsSNPs) on carriers. I believe predicting protein-specific functional phenotypes accurately of nsSNPs within these genes is the best path to reaching this goal. My initial efforts will focus on the ion channels associated with LQT1, LQT2, LQT3, and LQT5.

								For the last ten years, I have developed experimental and computational methodologies to study membrane protein structure and flexibility. At the University of Virginia, I focused on the techniques of electron paramagnetic resonance (EPR) and crystallography. At Vanderbilt, I have focused on investigations of membrane proteins involved in human disease, including the KCNQ and KCNE potassium channel α and β subunits. The last two years of my training focused on learning from experts in the fields of cardiac arrhythmia genetics (Dan Roden), and computational phenotype-predictive modeling (Jens Meiler and Jeffrey Blume) to construct predictive models of ion channel phenotypes and validate the resulting predictions.</p>
								<header class="align-center">
									<h3>Perturbation of ion channel function and relationship to disease</h3>
								</header>
								<div class="row 200%">
									<div class="6u 12u$(medium)">
										<p>Current prediction algorithms fail to predict the effects of nsSNPs due in part to the inability of models to account for the many factors that prevent complete penetrance—noise from complicating genetic and environmental risk factors buries the protein-specific functional signal. To help illustrate this point I curated a set of nearly 1,400 nsSNPs from the available literature on SCN5A, including 304 variants that were at least partially functionally characterized. What I learned can be summarized as follows:
										There is a range of tolerated NaV1.5 perturbation. There is also a range of perturbation that is not well tolerated. In both these extremes, clinical presentation is largely homogeneous. However, modest perturbations result in varied clinical presentations that are heterogeneous. This trend holds when comparing among different variants, but also holds within the population of carriers of a single variant (Table 1).
										I believe predicting SCN5A variant function will be more informative to an accurate risk of clinical presentation, probabilistically, and will enable the more consistent use of in silico methods in clinical diagnosis.</p>
									</div>
										<h5>Table 1. Example of intravariant (hetero/homo)geneity of clinical presentation<h5>
											<table>
												<thead>
													<tr>
														<th>Variant</th>
														<th>Peak Current[1]</th>
														<th># Unaffected[2]</th>
														<th># BrS1[3]</th>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>S1787N</td>
														<td>95%</td>
														<td>12</td>
														<td>1</td>
													</tr>
													<tr>
														<td>Y1795H</td>
														<td>66%</td>
														<td>7</td>
														<td>5</td>
													</tr>
													<tr>
														<td>R367H</td>
														<td>0%</td>
														<td>3</td>
														<td>16</td>
													</tr>
												</tbody>
											</table>
													1. A proxy for channel function
													2. Number of carriers without a clinical phenotype
													3. Number of carriers diagnosed with BrS1
									</div>
										<header class="align-center">
								<h3>Predicting ion channel function</h3>
								</header>
							<p>My lab aims to address the challenge of VUSs by blending experimental and computational strategies: 1) determine structure and flexibility-induced changes from selected SCN5A variants using a combination of Rosetta modeling, molecular dynamics in Amber, and nuclear magnetic resonance (NMR). 2) generate experimental deep mutational scanning data sensitive to trafficking/functionally defective KCNH2 and KCNQ1-KCNE1 variants. 3) functionally characterize variants in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). These technologies will enable me to construct predictive models of ion channel phenotypes and validate the resulting predictions.</p>
							<p><b>Computational and experimental structural biology.</b> The ultimate goal of generating phenotypes of any ion channel nsSNP is only currently possible in silico. The current ability to computationally predict protein structure is still a developing field, especially for membrane proteins (3, 4); predicting membrane protein flexibility ab initio is even less developed. To help guide structural modeling of NaV1.5 efforts, my lab will experimentally structurally characterize the voltage-sensing module of Domain IV with NMR. Further, my lab will develop fully solvated, full-atom trajectories of NaV1.5 for comparison with NMR experimental data and for high-throughput variant flexibility determination in silico. The advantage of this approach is the capability of more high-throughput parallel running of variants in silico than could be concievably determined experimentally.</p>
							<p><b>Deep mutational scanning</b>. Computational approaches will never eclipse the need for experimental data of some resolution. For example, although it has been known for years that destabilization-induced mistrafficking is the most common underlying cause for loss-of-function variants of proteins in general (5), computationally predicting changes in membrane protein stability due to missense mutations remains an unsolved challenge (2). To complement large-scale variant characterization done computationally, my lab will assay functionality of all codon substitutions within targeted 100 residue segments in KCNH2 and KCNQ1-KCNE1, genes implicated in LQT1, LQT2, and LQT5. A modified HEK cell line generated by the Fowler lab at the University of Washington and shared with the Roden lab at Vanderbilt enables very high-throughput analysis of variants though at relatively low resolution (6). The cell line allows for the incorporation of a single plasmid at a specific locus in an individual HEK cell genome which can then be sequenced and quantitated following a selection assay.</p>
							<p><b>hiPSC-CM.</b> The relationship between function of SCN5A variants characterized by heterologous expression and resulting action potential perturbations in people is not always straight forward. Though the there is a clear correlation between SCN5A variant perturbation measured by heterologous expression and clinical presentation, there are variants where relatively high minor allele frequency (MAF, as measured by gnomAD) conflicts with the severity of functional perturbation. This begs the question of whether the degree of perturbation is tolerable or whether the heterologous expression data are not representative of the true phenotypes in human cardiomyocytes. My lab will test SCN5A variant phenotypes in the hiPSC-CM model system. This model system allows for a direct assessment of the variant-specific perturbation against isogenic controls. The process takes approximately 30-40 days for cells to reach maturity. In the best-case scenario, running all variants in parallel, genome editing, differentiation into myocytes, and electrophysiological characterization can be completed in three months. </p>
						</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<footer id="footer">
				<div class="container">
					<ul class="icons">
						<li><a href="#" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
						<li><a href="#" class="icon fa-instagram"><span class="label">Instagram</span></a></li>
						<li><a href="#" class="icon fa-envelope-o"><span class="label">Email</span></a></li>
					</ul>
				</div>
				<div class="copyright">
					&copy; Untitled. All rights reserved.
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
