<!DOCTYPE HTML>
<!--
	Hielo by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>Research Projects</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="subpage">

		<!-- Header -->
			<header id="header">
				<div class="logo"><a href="index.html">Kroncke Laboratory</a></div>
				<a href="#menu">Menu</a>
			</header>

			<!-- Nav -->
				<nav id="menu">
					<ul class="links">
						<li><a href="index.html">Home</a></li>
						<li><a href="ResearchProjects.html">Research Projects</a></li>
						<li><a href="People.html">People</a></li>
						<li><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=Kroncke+B*" target="_blank">Publications</a></li>
						<li><a href="Resources.html">Resources</a></li>
						<li><a href="OpenPositions.html">Open Positions</a></li>
					</ul>
				</nav>

		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>How do we determine the significance of genetic variants?</p>
						<h2>Research Program</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
			<section id="two" class="wrapper style2">
				<div class="inner">
					<div class="box">
						<div class="content">
							<header class="align-center">
								<h2>What is the significance of genetic variants to those that carry them?</h2>
							</header>
							The increasing use of next generation sequencing in the clinical arena is uncovering a large number of variants across all genes, but methods to predict their pathogenicity are not well established. This results in an increasing number of “variants of uncertain significance” (VUSs), a major emerging problem in genomic medicine. Rare variants (mutations) in the cardiac ion channels are implicated in diverse heart diseases, including, long QT syndrome (LQTS), short QT syndrome (SQTS), and Brugada syndrome (BrS), but are also common in healthy populations. While multiple algorithms predict whether ion channel variants are deleterious (SIFT, PolyPhen-2, PredSNP, CADD, etc.), too many neutral variants are classified as disease-causing. The long-term research interest of the laboratory is to improve our understanding of the clinical burden of ion channel non-synonymous single nucleotide polymorphisms (nsSNPs) on carriers. I believe predicting protein-specific functional phenotypes accurately of nsSNPs within these genes is the best path to reaching this goal. Our initial focus is on ion channels associated with Long QT Syndrome.</p>
								<header class="align-center">
									<h3>Perturbation of ion channel function and relationship to disease</h3>
								</header>
								<div class="row 200%">
									<div class="6u 12u$(medium)">
										<p>Current prediction algorithms fail to predict the effects of nsSNPs due in part to the inability of models to account for the many factors that prevent complete penetrance—noise from complicating genetic and environmental risk factors buries the protein-specific functional signal. To help illustrate this point I curated a set of nearly 1,400 nsSNPs from the available literature on SCN5A, including 304 variants that were at least partially functionally characterized. The results can be summarized as follows:
										There is a range of tolerated NaV1.5 perturbation. There is also a range of perturbation that is not well tolerated. In both these extremes, clinical presentation is largely homogeneous. However, modest perturbations result in varied clinical presentations that are heterogeneous. This trend holds when comparing among different variants, but also holds within the population of carriers of a single variant (Table 1).
										This highlights that SCN5A variant function is informative to an accurate risk of clinical presentation, probabilistically, and <i>in silico</i> models that can accurately predict changes in function may be of greater utility in clinical diagnosis than models that only predict binary "pathogenic" or "benign".</p>
									</div>
										<div class="6u$ 12u$(medium)">
										<h5>Table 1. Example of intravariant (hetero/homo)geneity of clinical presentation<h5>
											<table>
												<thead>
													<tr>
														<th>Variant</th>
														<th>Peak Current[1]</th>
														<th># Unaffected[2]</th>
														<th># BrS1[3]</th>
													</tr>
												</thead>
												<tbody>
													<tr>
														<td>S1787N</td>
														<td>95%</td>
														<td>12</td>
														<td>1</td>
													</tr>
													<tr>
														<td>Y1795H</td>
														<td>66%</td>
														<td>7</td>
														<td>5</td>
													</tr>
													<tr>
														<td>R367H</td>
														<td>0%</td>
														<td>3</td>
														<td>16</td>
													</tr>
												</tbody>
											</table>
													[1] A proxy for channel function
													[2] Number of carriers without a clinical phenotype
													[3] Number of carriers diagnosed with BrS1
									</div>
								</div>
										<header class="align-center">
											<h3>Predicting ion channel function</h3>
										</header>
							<p>My laboratory addresses the challenge of VUSs by blending experimental and computational strategies: 1) structure and flexibility-induced changes from selected ion channel variants using a combination of Rosetta modeling, molecular dynamics in AMBER, and nuclear magnetic resonance (NMR) 2) experimental deep mutational scanning data sensitive to trafficking/functionally defective ion channel variants. 3) functional effects of variants in human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). These technologies enable us to construct predictive models of ion channel phenotypes and validate the resulting predictions.</p>
							<p><b>Computational and experimental structural biology.</b> The ultimate goal of generating phenotypes of any ion channel nsSNP is only currently possible <i>in silico</i>. My laboratory experimentally structurally characterizes domains of ion channels using NMR. Further, we develop fully solvated, full-atom trajectories of ion channels for comparison with NMR experimental data and for high-throughput variant flexibility determination <i>in silico</i>. The advantage of this approach is the capability of more high-throughput parallel running of variants <i>in silico</i> than could be concievably determined experimentally.</p>
							<p><b>Deep mutational scanning</b>. Computational approaches will never eclipse the need for experimental data of some resolution. To complement large-scale variant characterization done computationally, my laboratory assays functionality of all codon substitutions within targeted segments in LQTS-implicated ion channels. This technology enables very high-throughput analysis of variants at relatively low resolution.</p>
							<p><b>hiPSC-CM.</b> The relationship between function of ion channel variants characterized by heterologous expression and resulting action potential perturbations in people is not always straight forward. Though the there is a clear correlation between SCN5A variant perturbation measured by heterologous expression and clinical presentation, there are variants where relatively high minor allele frequency (MAF, as measured by gnomAD) conflicts with the severity of functional perturbation. This begs the question of whether the degree of perturbation is tolerable or whether the heterologous expression data are not representative of the true phenotypes in human cardiomyocytes. My laboratory tests ion channel variant phenotypes in the hiPSC-CM model system. This model system allows for a direct assessment of the variant-specific perturbation against isogenic controls.</p>

							<header class="align-center">
								<h3>Funding</h3>
							</header>
							<p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p>
							<span class="image left"><img  src="images/NHLBI.png"  alt="" /></span>
							<span class="image center"><img  src="images/DHHS.png"  alt="" /></span>
							<span class="image right"><img  src="images/VUMC.png"  alt="" /></span><p><p>
							<p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p><p>
							<p>This work is funded by National Institutes of Health grant R00 HL135442 and Vanderbilt University Medical Center.
							</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<footer id="footer">
				<div class="container">
					<ul class="icons">
						<li><a href="People.html" class="icon fa-envelope-o"><span class="label">Email</span></a></li>
					</ul>
				</div>
				Kroncke Laboratory<br>Vanderbilt University Medical Center<br>
			Vanderbilt Center for Arrhythmia Research and Therapeutics (VanCART)<br>1225E MRBIV<br>2215B Garland Ave.<br>Nashville, TN 37232<br>Email: brett.kroncke@gmail.com
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
